test/junit/scala/collection/immutable/LazyListTest.scala

```diff
@@ -340,28 +340,22 @@ class LazyListTest {
 
     def precyc(n: Int, m: Int) = pre(n) #::: cyc(m)
 
-    def str(s: Stream[Int]) = {
-      val b = new StringBuilder
-      s.addString(b, "", "", "")
-      b.toString
-    }
-
     def goal(n: Int, m: Int) = (-n until m).mkString + "..."
 
     // Check un-forced cyclic and non-cyclic streams
-    assertEquals(pre(2).take(1).toList.mkString + "?", str(pre(2)))
-    assertEquals(cyc(2).take(1).toList.mkString + "?", str(cyc(2)))
-    assertEquals(precyc(2,2).take(1).toList.mkString + "?", str(precyc(2,2)))
+    assertEquals("Stream(-2, ?)", pre(2).toString)
```

Comment:
the results of these tests didn't change, but I'm making the expectations explicit.